uber/menu.py

```diff
@@ -51,27 +51,26 @@ def render_items_filtered_by_current_access(self):
         """
         out = {}
 
-        allowed = True
-        if self.access:
-            allowed = False
-            for access_level in listify(self.access):
-                if access_level in sa.AdminAccount.access_set():
-                    allowed = True
-                    break
-
-        if not allowed:
+        if self.access and set(listify(self.access)).isdisjoint(sa.AdminAccount.access_set()):
             return None
 
         out['name'] = self.name
         if self.submenu:
             out['submenu'] = []
-            for submenu in self.submenu:
-                out['submenu'].append(submenu.render_items_filtered_by_current_access())
+            for menu_item in self.submenu:
+                filtered_menu_items = menu_item.render_items_filtered_by_current_access()
+                if filtered_menu_items:
+                    out['submenu'].append(filtered_menu_items)
         else:
             out['href'] = self.href
 
         return out
 
+    def __getitem__(self, key):
+        for sm in self.submenu:
+            if sm.name == key:
+                return sm
+
 
```

Comment:
this ```__getitem__``` is optional but useful for plugins to be able to say stuff like this:
```
c.MENU['People'].append_menu_item(
    MenuItem(access=c.BANDS, name='Bands', href='../band_admin/')
)
```

some open PRs are in the plugins here:
https://github.com/magfest/bands/pull/45